backend/app/crud/assistants.py

```
@@ -1,32 +1,57 @@
+    project_id: int,
+    skip: int = 0,
+    limit: int = 100,
+) -> list[Assistant]:
+    """
+    Return all assistants for a given project and organization, with optional pagination.
+    """
+    statement = (
+        select(Assistant)
+        .where(
+            Assistant.project_id == project_id,
+            Assistant.is_deleted == False,
```

Ruff (0.12.2)

Avoid equality comparisons to `False`; use `not Assistant.is_deleted:` for false checks

Replace with `not Assistant.is_deleted`

(E712)